.travis.yml

```diff
@@ -22,8 +22,10 @@ matrix:
             - set -x
             # Make sure there are no left-over git patch markers in
             - git --no-pager grep '>>>>>>>'
-            - pip install -r requirements/linting.txt
-            - flake8 kuma docs tests
+            - pip install poetry~=1.0.0b8
```

Comment:
Can you switch to https://pypi.org/project/poetry/1.0.0b9/ so that when we start we're on the latest and greatest to start.